ert_data/loader.py

```diff
@@ -241,10 +241,16 @@ def _load_general_obs(facade, observation_keys, case_name):
             index = _create_multi_index(index_list, index_list)
 
             data.append(
-                pd.DataFrame(
-                    [node.get_data_points()], columns=index, index=["OBS"]
-                ).append(pd.DataFrame([node.get_std()], columns=index, index=["STD"]))
+                pd.concat(
+                    [
+                        pd.DataFrame(
+                            [node.get_data_points()], columns=index, index=["OBS"]
+                        ),
+                        pd.DataFrame([node.get_std()], columns=index, index=["STD"]),
+                    ]
+                )
```

Comment:
Since there already is a `pd.concat` on line 254, would it work to just append each of these frames to `data`?

Reply:
Good point, and yes I think we can.
I've also removed the `observations` list as it seems redundant.

Reply:
Tried making bigger changes, but tests are failing and I don't think it's worth it spending too much time on this.